test/test_city.py

```diff
@@ -65,7 +65,6 @@ def test_add_connection(self):
         self.city.add_connection(another_city)
         self.assertIn(another_city, self.city.connected_cities)
 
-    def test_get_cubes(self):
-        self.city.cubes['Blue'] = 2
-        self.assertEqual(2, self.city.cubes['Blue'])
-
+    def test_get_infection_levels(self):
```

Comment:
I think we can remove this method as useless (I wonder how it lived).

Reply:
I had assumed it to be out of the PR's scope. I have removed the test.